.travis/after_success_git.sh

```diff
@@ -24,6 +24,10 @@ if [ -n "$GH_TOKEN" ]; then
 		git -c user.name="travis" -c user.email="travis" commit -m "test(ci): update code style outputs"
 		echo "✓ Commit updated lint output to $TRAVIS_PULL_REQUEST_BRANCH"
 
+		git add --all
+		git -c user.name="travis" -c user.email="travis" commit -m "test(ci): prettier"
+		echo "✓ Commit prettified files to $TRAVIS_PULL_REQUEST_BRANCH"
+
```

Comment:
everything else is prettier ... not sure this is very futur proof may be just src ?

Reply:
:ok_hand: